test/e2e-common.sh

```diff
@@ -210,6 +216,11 @@ function install_knative_serving_standard() {
   echo ">> Bringing up Serving"
   kubectl apply -f "${INSTALL_RELEASE_YAML}" || return 1
 
+  if ((RECONCILE_GATEWAY)); then
```

Comment:
are you sure it works without space?

Reply:
yes. it works. But to keep consistent with the code in this file, I just added the space here.